stats/stats-server/tests/counters.rs

```diff
@@ -0,0 +1,75 @@
+use reqwest_middleware::{ClientBuilder, ClientWithMiddleware};
+use reqwest_retry::{policies::ExponentialBackoff, RetryTransientMiddleware};
+use stats::tests::{init_db::init_db_all, mock_blockscout::fill_mock_blockscout_data};
+use stats_server::{stats, Settings};
+use std::collections::HashSet;
+
+fn client() -> ClientWithMiddleware {
```

Comment:
Why do you need retry in tests?

Reply:
Yes, as we did in smart-contract-verifier-http. Server may not be started yet